spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,29 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+
+  context "#get_cis_all" do
+    elem_objects = [:storage, :miq_enterprise, :ext_management_system, :ems_cluster, :tenant]
+    elem_objects.each do |obj|
+      let!(obj) { FactoryGirl.create(obj) }
+    end
+    elements_miq = %w(enterprise storage ext_management_system ems_cluster tenant)
+    elements_miq.each do |element|
+      classtype =
+        if element == "enterprise"
+          MiqEnterprise
+        else
+          element.classify.constantize
+        end
```

Comment:
@lpichler I'm pretty sure it refers to "Configuration Items". Those would be objects that are managed by a configuration management system like ours. They would include Vms, Hosts, Clusters, Storages, etc.